### PR TITLE
Incomplete: Swaroop: Task689-692_MLQA_Gutenberg_time

### PR DESCRIPTION
**Pull request by Ratnam-Parikh**

### CODE REVIEW

**Comment by swarooprm** [comment]
@Ratnam-Parikh, you have updated 2 tasks here, do you plan to add more or we will start reviewing?

**Comment by Ratnam-Parikh** [comment]
I do plan to add more, I will do it shortly.

**Comment by Ratnam-Parikh** [comment]
> @Ratnam-Parikh, you have updated 2 tasks here, do you plan to add more or we will start reviewing?

I will add more tasks shortly.

**Comment by swarooprm** [comment]
@nrjvarshney, wondering if you can help review these tasks?

**Comment by nrjvarshney** [comment]
Task 689-
* "input_language", "output_language" fields are missing. refer to the json format instructions


Task 690 - 
* "The title should be generated using the given input only." seems redundant
* improve the explanations


Task 692 - 
* ". The output should be in words for the time referenced in the input." This does not follow the examples and instances

**Comment by Ratnam-Parikh** [comment]
> Task 689-
> 
> * "input_language", "output_language" fields are missing. refer to the json format instructions
> 
> Task 690 -
> 
> * "The title should be generated using the given input only." seems redundant
> * improve the explanations
> 
> Task 692 -
> 
> * ". The output should be in words for the time referenced in the input." This does not follow the examples and instances

Yes, I will upload the changes shortly.

File: tasks/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ## Tasks List 
 
-This directory contains the tasks that are part of this benchmark. 
+This directory contains the tasks that are part of this benchmark.
 
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```
Comment (nrjvarshney) [inline tasks/README.md:9]: why did you change this?

Comment (Ratnam-Parikh) [reply tasks/README.md:9]: I didn't change the default tasks uploaded in the repo.

**Comment by nrjvarshney** [comment]
I don't understand tasks 691, 692 --> do they even make sense?

**Comment by Ratnam-Parikh** [comment]
> I don't understand tasks 691, 692 --> do they even make sense?

These tasks are to extract time referenced in the given input.

For example:

Input: "So profoundly penetrated with thoughtfulness was the tone of his voice that I could not take umbrage . The attempt to analyze his signification cost me an aching forehead , perhaps because I knew it too acutely . She was on horseback ; I on foot , Schwartz for sole witness , and a wide space of rolling silent white country around us . We had met in the fall of the winter noon by accident . ` You like my Professor ? ' said Ottilia . ` I do : I respect him for his learning . '"
Output: "the fall of the winter noon"

In the input, the author is using "the fall of the winter noon" as the time and the task is to extract such phrases or specific time references.

**Comment by swarooprm** [comment]
@Ratnam-Parikh, I understand the reviewer's concerns with tasks 691 and 692. I suggest you to create some more translation tasks since you have lots of such options available in MLQA with data in multiple languages. Let me know once you are done and I will evaluate your HW.

**Comment by Ratnam-Parikh** [comment]
> @Ratnam-Parikh, I understand the reviewer's concerns with tasks 691 and 692. I suggest you to create some more translation tasks since you have lots of such options available in MLQA with data in multiple languages. Let me know once you are done and I will evaluate your HW.

Hi, 

I wanted to let you know that I have various other deadlines currently, and thus creating more tasks will not be possible from my side. Also, the tasks 691 and 692 do make sense for the dataset they are created for, as they are related to text summarization and extraction. I hope you consider the situation and grade accordingly.

Thank you.

**Comment by nrjvarshney** [comment]
@Ratnam-Parikh , other issues with your tasks:
* If you look at the first example then it mentions "noon" as well but it is not there in the output.
* Several outputs have extra '\n'.
* "the fall of the winter noon" how to decide whether it should be just "noon", "winter noon", "fall of the winter noon", or "the fall of the winter noon"?

**Comment by Ratnam-Parikh** [comment]
> @Ratnam-Parikh , other issues with your tasks:
> 
> * If you look at the first example then it mentions "noon" as well but it is not there in the output.
> * Several outputs have extra '\n'.
> * "the fall of the winter noon" how to decide whether it should be just "noon", "winter noon", "fall of the winter noon", or "the fall of the winter noon"?

Yes, the dataset has the column time reference and it has only one value for every input and that is “the fall of the winter noon” and so in extraction from dataset I have one value in output.
